functions/parse-document/1.1/function.json

```diff
@@ -0,0 +1,69 @@
+{
+  "description": "Extract and convert textual information from various document formats, including docx, pptx, xlsx, odt, odp, ods, and pdf files.",
+  "label": "Parse Document to Text",
+  "category": "Misc",
+  "icon": {
+    "name": "FileIcon",
+    "color": "Orange"
+  },
+  "options": [
+    {
+      "meta": {
+        "type": "Value",
+        "allowedKinds": ["STRING", "FILE"],
+        "validations": {
+          "required": true
+        }
+      },
+      "name": "document",
+      "label": "Document",
+      "info": "Specify the document to be parsed. This can be a file property or an URL."
+    },
+    {
+      "meta": {
+        "type": "Number",
+        "default": 300,
+        "validations": {
+          "required": true
+        }
+      },
+      "advanced": true,
+      "name": "density",
+      "label": "Density",
+      "info": "This option specifies the image resolution. The higher the density, the better the quality of the output will be. However, higher density also means slower processing."
+    },
+    {
+      "meta": {
+        "type": "Boolean"
+      },
+      "advanced": true,
+      "name": "forceImage",
+      "label": "Force image",
+      "info": "Force the document to be scanned as an image. Sometimes this can result in a better output."
+    },
+    {
+      "meta": {
+        "type": "Boolean",
+        "default": true
+      },
+      "advanced": true,
+      "name": "removeSpecialCharacters",
+      "label": "Remove Special Characters",
+      "info": "Remove special characters from the output."
```

Comment:
Could be clearer about what the option does. Decoding html-encoded spaces

Reply:
updated